components/classes/b2b/demandbase-account-intent.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/demandbase-account-intent",
```

Comment:
When you move the file to the other location, please change the $id to `https://ns.adobe.com/b2b/demandbase/classes/demandbase-account-intent`